src/static/js/client/sidebar-search.js

```diff
@@ -344,8 +344,8 @@ export function addPageListeners() {
 
   info.searchInput.addEventListener('keydown', domEvent => {
     if (domEvent.key === 'ArrowDown') {
-      const elem = info.results?.firstChild;
-      if (elem && !elem.classList.contains('wiki-search-no-results')) {
+      const elem = info.results.firstChild;
+      if (elem?.classList?.contains?.('wiki-search-result')) {
```

Comment:
I CHANGED MY MIND EHEEEEEE
```suggestion
      if (elem && elem.classList.contains('wiki-search-result')) {
```